### DISCUSSION TITLE
checking the availability steam market

### DISCUSSION THREAD

**Discussion by evgewakan** [chr233/ASFEnhance#188] [功能建议 / Ideas]
If don’t buy anything on steam market for a year or in a game, then steam market will be closed until you top up your balance / buy the game

You can add a command to check the availability of the trading platform (status is not suitable in this case)? 

https://help.steampowered.com/en/faqs/view/451E-96B3-D194-50FC#nottrusted
![123](https://github.com/chr233/ASFEnhance/assets/74374112/cbed2691-a1c1-4f99-9eb1-32310f819f42)

**Comment by chr233** [comment]
added
